src/ngraph/builder/split.hpp

```diff
@@ -50,5 +50,43 @@ namespace ngraph
         /// \return     The vector containing multiple nodes we split input node into.
         ///
         NodeVector split(const Output<Node>& value, size_t split_parts, int axis = 0);
+
+        namespace opset1
+        {
+            /// \brief      Split value on specified axis into multiple parts.
+            ///
+            /// \param[in]  value         The value to be split.
+            /// \param[in]  length_parts  The vector defining the lengths of each split part.
+            /// \param[in]  axis          The axis we split input node on. Default value is zero
+            ///                           axis.
+            /// \note       This implementation supports negative `axis` values (similar to NumPy
+            ///             indexing). This means that the axis to split on will be counted from
+            ///             the back of the tensor (negative values are subtracted from its rank).
+            ///
+            /// \return     The vector containing multiple nodes we split input node into.
+            ///             The vector is output of Split:v1 op
+            ///
+            NodeVector split(const Output<Node>& value,
+                             const std::vector<size_t>& length_parts,
+                             int64_t axis = 0);
+
+            /// \brief      Split node on specified axis into multiple parts.
+            ///
+            /// \param[in]  value         The value to split.
+            /// \param[in]  split_parts   The number of parts we want to split output at given
+            ///                           axis. The length of the axis to split must be divisible by
+            ///                           this value.
+            /// \param[in]  axis          The axis we split input node on. Default value is zero
+            ///                           axis.
+            ///
+            /// \note       This implementation supports negative `axis` values (similar to NumPy
+            ///             indexing). This means that the axis to split on will be counted from
+            ///             the back of the tensor (negative values are subtracted from its rank).
+            ///
+            /// \return     The vector containing multiple nodes we split input node into.
+            ///             The vector is output of VariadicSplit:v1 op
+            ///
+            NodeVector split(const Output<Node>& value, size_t split_parts, int64_t axis = 0);
```

Comment:
Could you use attribute name from the spec?
```suggestion
            NodeVector split(const Output<Node>& value, size_t num_splits, int64_t axis = 0);
```

Reply:
done, used `num_splits` to `Split`, and `split_lengths` to `VariadicSplit`